pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : str, numpy.dtype, or Python dtype, default None
```

Comment:
@jorisvandenbossche : fair enough, though that isn't entirely what @jreback was asking for.  In any case, I'm going to refrain from making any further changes until @jreback can provide further clarification.